seqr/views/react_app_tests.py

```diff
@@ -79,21 +80,22 @@ def test_no_login_react_page(self):
         response = self.client.get(url)
         self._check_page_html(response, 'test_user')
 
+    @mock.patch('seqr.utils.search.elasticsearch.es_utils.ELASTICSEARCH_SERVICE_HOSTNAME', 'testhost')
     @mock.patch('seqr.views.react_app.ANVIL_LOADING_DELAY_EMAIL_START_DATE', '2022-12-01')
     @mock.patch('seqr.views.react_app.datetime')
-    def test_react_page_anvil_loading_delay(self, mock_datetime):
+    def test_react_page_additional_configs(self, mock_datetime):
```

Comment:
Should we add a test without the `ELASTICSEARCH_SERVICE_HOSTNAME` configuration?

Reply:
Thats the default, every other test in this file is testing that